apps/wallet/src/ui/app/hooks/useOwnedNFT.ts

```diff
@@ -15,7 +15,7 @@ export function useOwnedNFT(
     nftObjectId: string | null,
     address: SuiAddress | null
 ) {
-    const data = useGetObject(nftObjectId!);
+    const data = useGetObject(nftObjectId || '', { showDisplay: true });
```

Comment:
I feel like this is easy enough to get wrong that I'd prefer to just get all the data in the hook always.